src/vue/fields/DateField.vue

```diff
@@ -38,23 +35,47 @@
 </template>
 
 <script>
-import FlatPickr from 'vue-flatpickr-component'
+import Flatpickr from 'flatpickr'
 import moment from 'moment'
 
-const EVENTS = {
+// All supported events by Flatpickr
+const ALLOWED_FLATPICKR_EVENTS = {
```

Comment:
`ALLOWED_` excess here
also looks like `_HOOKS` not events

Reply:
`HOOKS` means a callback of the event rather but not event name. So `EVENTS` looks better, as I think

Reply:
to hook